client/js/components/map/admin/MapAdmin.vue

```diff
@@ -82,42 +82,49 @@
       }" />
 
     <div class="layout__item u-1-of-1 text-right u-mt-0_5 space-inline-s">
-      <input
-        class="btn btn--primary"
+      <dp-button
+        data-cy="save"
         :disabled="!areScalesSuitable"
-        type="submit"
         name="saveConfig"
-        :value="Translator.trans('save')"
-        @click="() => save()">
-      <input
+        :text="Translator.trans('save')"
+        type="submit"
+        @click="() => save()" />
+      <dp-button
         v-if="hasPermission('area_admin_single_document')"
-        class="btn btn--primary"
+        data-cy="saveAndReturn"
         :disabled="!areScalesSuitable"
-        type="submit"
         name="submit_item_return_button"
-        :value="Translator.trans('save.and.return.to.list')"
-        @click="() => save(true)">
-      <a
-        class="btn btn--secondary"
-        :href="Routing.generate('DemosPlan_element_administration', { procedure: procedureId })">
-        {{ Translator.trans('abort') }}
-      </a>
+        :text="Translator.trans('save.and.return.to.list')"
+        type="submit"
+        @click="() => save(true)" />
+      <dp-button
+        color="secondary"
+        data-cy="abort"
+        :href="Routing.generate('DemosPlan_element_administration', { procedure: procedureId })"
+        :text="Translator.trans('abort')" />
+      <dp-button
+        color="secondary"
+        data-cy="reset"
+        :text="Translator.trans('reset')"
+        variant="outline"
```

Comment:
I never documented it but within a button group, buttons should only differenciate by color, not by variant.

Reply:
all right 46713c14e5376654601d7d0c02b140fccbf25677